Test3/Tests/UnitTest1.cs

```diff
@@ -0,0 +1,42 @@
+using Test3;
+using NUnit.Framework;
+
+namespace Tests
+{
+  
+    public class ThreadSafeBlockingQueueTest
+    {
+        
+        [Test]
+        public void DequeueTest()
+        {
+            var queueTest = new ThreadSafeBlockingQueue<string>();
+            queueTest.Enqueue("Hello", 1);
+            queueTest.Enqueue("World", 2);
+            var res = queueTest.Dequeue();
+            Assert.AreEqual("World", res);
+
+        }
```

Comment:
```suggestion
            Assert.AreEqual("World", res);
        }
```
Перед закрывающей фигурной скобкой пустая строка не ставится